vladiate/examples/bats.csv

```diff
@@ -0,0 +1,4 @@
+Column A|Column B
+Vlad the Impaler|Not A Vampire
+Dracula|Vampire
+Count Chocula|Vampire
```

Comment:
Can you add a `Vlad` subclass to `vladiate/vladiate/examples/vladfile.py` which uses this file and the delimiter as well?